### PR TITLE
Create tower_inventory_refresh.py awx_collection

### PR DESCRIPTION
**Pull request by chris93111**
##### SUMMARY
This module update all sources in inventory

-->

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
awx_collection

##### AWX VERSION
```
13.0.0
```

### CODE REVIEW

File: awx_collection/plugins/modules/tower_inventory_refresh.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+short_description: create, update, or destroy Ansible Tower inventory.
+description:
+    - Refresh all source in inventory
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    name:
+      description:
+        - The name to use for the inventory.
+      required: True
+      type: str
```
Comment (AlanCoding) [inline awx_collection/plugins/modules/tower_inventory_refresh.py:29]: Since inventory name is not unique, this should take `organization`

**Comment by AlanCoding** [comment]
There is no module to update an individual inventory source.... with that not present, I can see the case for this.

File: awx_collection/plugins/modules/tower_inventory_refresh.py (hunk 1 above)
Comment (chris93111) [reply awx_collection/plugins/modules/tower_inventory_refresh.py:29]: ok i have add organization to option and put control with get_one

**Comment by AlanCoding** [comment]
looks like failures are due to a refactor that happened before you rebased, with some name changes

**Comment by chris93111** [comment]
@AlanCoding 

ok it's apply

**Comment by AlanCoding** [comment]
I want to point out that examples in #8033 offer

```yaml
- name: Update all inventory sources
  tower_inventory_source_update:
    inventory: "My Other Inventory"
    inventory_source: "{{ item }}"
  loop: "{{ query('awx.awx.tower_api', 'inventory_sources', query_params={ 'inventory': 30 }, return_ids=True ) }}"
```

That doesn't use the API endpoint this module does, but it's a very maintainable pattern that gets all the functionality.

**Comment by AlanCoding** [comment]
We appreciate the contribution, but want a more serious re-consideration in light of https://github.com/ansible/awx/pull/8033 merging, and I'd prefer to document the loop instead of a new module. Maybe a role to get reuse?

**Pull request closed without merging**